prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-table/site-registration-table.component.html

```diff
@@ -0,0 +1,111 @@
+<div class="row">
+  <div class="col">
+
+    <table mat-table
+           [dataSource]="dataSource"
+           class="w-100">
+
+      <!-- TODO change in requirements dropped organization -->
```

Comment:
Note for reviewers. Changes in requirements dropped using organizations so commented out, and kept until fruitloops next sprint